fiatconnect-api.md

```diff
@@ -1796,6 +1796,897 @@ is below:
 
 * `'NG'`: The account number should be exactly 10 digits long, and only include the numbers 0-9.
 
+####  7.3.2.2. `CheckingAccountIfsc`
+
+`CheckingAccountIfsc` is a Fiat Account schema that represents accounts for transactions across `INR` currency.
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "ifsc": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
+    "fiatAccountType": "CheckingAccountIfsc"
+}
+```
+####  7.3.2.3. `CheckingAccountIban`
+
+`CheckingAccountIban` is a Fiat Account schema that represents accounts for transactions across `EUR` currency.
+
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "iban": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
```

Comment:
iban actually includes a country code as the prefix

Reply:
though this field may still be useful so the provider can more easily use "allowedValues" to restrict which countries are allowed

Reply:
When providing a quote the rate could differ per country. 
Initially, we had the list of countries in allowed values format and as per @jophish comment, we changed this.